example/storybook/stories/TrackTile/util/MockEnvironmentDecorator.tsx

```diff
@@ -6,13 +6,13 @@ import {
   Tracker,
   TrackTileServiceProvider,
   useAxiosTrackTileService,
-} from '../../../../src/components/TrackTile/main';
+} from '../../../../../src/components/TrackTile/main';
```

Comment:
Opps. Thanks for fixing this, seems like there must be something we can do to allow an invalid path from being merged in.